.github/workflows/ci.yml

```diff
@@ -7,29 +7,22 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-24.04
 
     env:
       MIX_ENV: test
 
     strategy:
       # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#between-elixir-and-erlang-otp
       matrix:
-        os: [ubuntu-22.04, ubuntu-24.04]
```

Comment:
Erlang / Elixir don't depend on the OS. Also this was overwriting the caches.

Reply:
I am well aware of this , but was nice to have predictable LTS. I can remove this

Reply:
I know they dont, but getting predictable LTS helps sometime . I have had compiling issue in the past still for nif and stuff